internal/app/connectconformance/testsuites/data/connect_client_http_to_connect_code.yaml

```diff
@@ -1,15 +1,15 @@
 name: HTTP to Connect Code Mapping
-mode: TEST_MODE_CLIENT
-relevantProtocols:
-  - PROTOCOL_CONNECT
 # If a Connect server returns a non-200 HTTP code for unary requests without an
 # explicit Connect error code, the client must synthesize the HTTP code to a 
 # Connect code. These tests cases verify that mapping by forcing the server to
 # return a specified HTTP code and then test whether the client correct returns
 # the required Connect code.
+mode: TEST_MODE_CLIENT
+relevantProtocols:
+  - PROTOCOL_CONNECT
 testCases:
 - request:
-    testName: bad request
+    testName: unary/bad-request
```

Comment:
Ditto again. Everything in here is a unary RPC, so the "unary/" prefix seems like a bit more ceremony than is needed.